bundles/http/Citrix/Citrix-Virtual-Apps-and-Desktops-service/7687284a-09a1-4494-b33c-f6bfa0932656.js

```diff
@@ -16,7 +16,7 @@ async function fullSync(params) {
 }
 
 async function incrementalSync(params) {
-    await Promise.all([syncMachines(params), syncSessions(params)])
+    await Promise.all([syncSessions(params)])
```

Comment:
`Promise.all` is used when you have more promises.

```suggestion
    await syncSessions(params)
```

or this can work too
```js
function incrementalSync(params) {
    return syncSessions(params)
}
```